cmd/kubeadm/app/cmd/init.go

```diff
@@ -226,6 +230,9 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiv1beta1.InitConfig
 	)
 	flagSet.StringVar(featureGatesString, "feature-gates", *featureGatesString, "A set of key=value pairs that describe feature gates for various features. "+
 		"Options are:\n"+strings.Join(features.KnownFeatures(&features.InitFeatureGates), "\n"))
+	flagSet.StringVar(
+		&cfg.ImageRepository, "image-repository", cfg.ImageRepository,
+		`Choose a container registry to pull control plane images from`)
```

Comment:
i think we should remove this flag. the image repository is already exposed via the config.
but also we have some new image related fields for the addons in 1.13.